CONTRIBUTING.rst

```diff
@@ -190,6 +190,11 @@ Running System Tests
    $ export CLOUDSDK_PYTHON_SITEPACKAGES=1
 
    # Authenticate the gcloud tool with your account.
+   $ P12_CREDENTIALS_FILE="path/to/app_credentials.json"
```

Comment:
It is only deprecated in the `gcloud` SDK / CLI, not deprecated to actually use them.

I haven't looked at this in awhile. I don't recall why I used P12 here instead of JSON.